src/encoder.c

```diff
@@ -493,6 +493,45 @@ static uint32_t encode_gtime_ber(void *valptr,
     return towrite + ret;
 }
 
+static uint32_t encode_utctime_ber(void *valptr,
+        uint32_t len, uint8_t *buf, uint32_t rem) {
+
+    struct timeval *tv = (struct timeval *)valptr;
```

Comment:
Let's move the common code from this function and `encode_utctime()` into an inline function so we have less code duplication.